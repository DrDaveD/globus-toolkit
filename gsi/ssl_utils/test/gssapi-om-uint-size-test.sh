#!/bin/sh
######################################################################
#
# gssapi-om-uint-size-test.sh
#
# Make sure sizeof(OM_uint32) is consistent and makes sense.
#
######################################################################

size1=`./gssapi-om-uint-size-test-1`
size2=`./gssapi-om-uint-size-test-2`

if test $size1 -ne $size2 ; then
	echo "gssapi-om-uint-size-test failed."
	echo "Sizes were: $size1 and $size2"
	exit 1
fi

echo "gssapi-om-uint-size-test passed."
exit 0