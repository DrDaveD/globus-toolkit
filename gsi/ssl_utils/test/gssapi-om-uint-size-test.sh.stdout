gssapi-om-uint-size-test passed.
